src/p2p/supersocket.erl

```diff
@@ -0,0 +1,34 @@
+%% this module contains a supervised socket server thingy
+-module(supersocket).
+-behaviour(supervisor).
+
+-export([start_socket_server/0]).
+-export([start_socket/0]).
+-export([init/1]).
+
+-include_lib("stdlib/include/assert.hrl").
+
+start_socket_server() ->
+  io:fwrite("starting socket server on port 5678\n"),
```

Comment:
why is the port# the same as my password ಠ_ಠ

Reply:
for security :o